lib/happo/server.rb

```diff
@@ -52,6 +52,49 @@ def h(text)
       }
     end
 
+    get '/review-demo' do
+      diff_images = [
+        {
+          description: 'First component',
+          viewport: 'small',
+          url: 'http://placehold.it/350x150',
+        },
+        {
+          description: 'First component',
+          viewport: 'medium',
+          url: 'http://placehold.it/550x150',
+        },
+        {
+          description: 'First component',
+          viewport: 'large',
+          url: 'http://placehold.it/850x150',
+        },
+      ]
+
+      new_images = [
+        {
+          description: 'New component',
+          viewport: 'small',
+          url: 'http://placehold.it/350x150',
+        },
+        {
+          description: 'New component',
+          viewport: 'medium',
+          url: 'http://placehold.it/550x150',
+        },
+        {
+          description: 'New component',
+          viewport: 'large',
+          url: 'http://placehold.it/850x150',
+        },
+      ]
```

Comment:
This is good for documenting the structure of the objects we pass to the template as well.